template.yaml

```diff
@@ -49,6 +49,12 @@ Parameters:
     Description: Email domain used by Synapse
     Default: '@synapse.org'
 
+  AdminEmail:
```

Comment:
can you add some info about this to the README?

Reply:
whoops, yeah, done